src/App.tsx

```diff
@@ -15,37 +18,92 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+const SORT_BY_LENGTH = 'length';
+const SORT_BY_ABC = 'abc';
```

Comment:
You can remove this variables and just write in 'enum' key and value. It's like:
`enum SortType { length = 'length' } `